install/resources/mybb_theme.xml

```diff
@@ -929,8 +929,7 @@ blockquote cite span.highlight {
 
 div.error {
 	padding: 5px 10px;
-	border-top: 2px solid #FFD324;
-	border-bottom: 2px solid #FFD324;
+	border: 2px solid #FFD324;
```

Comment:
The stylesheet `global.css` needs its version updating to `1811` for this change too.